account_fiscal_position_autodetect_optional_vies/README.rst

```diff
@@ -98,6 +102,6 @@ Current `maintainer <https://odoo-community.org/page/maintainer-role>`__:
 
 |maintainer-victoralmau| 
 
-This module is part of the `OCA/account-fiscal-rule <https://github.com/OCA/account-fiscal-rule/tree/15.0/account_fiscal_position_autodetect_optional_vies>`_ project on GitHub.
+This module is part of the `OCA/account-fiscal-rule <https://github.com/OCA/account-fiscal-rule/tree/16.0/account_fiscal_position_autodetect_optional_vies>`_ project on GitHub.
```

Comment:
Did you modified this manually? No need.
Same for all the other modifications `15` -> `16`.

Reply:
No, I exported it from a local environment.